packages/atlas-service/src/main.ts

```diff
@@ -623,7 +623,9 @@ export class AtlasService {
       }
     }
 
-    await this.maybeWaitForToken({ signal });
+    if (process.env.COMPASS_E2E_SKIP_ATLAS_SIGNIN !== 'true') {
```

Comment:
I don't think this should be required here if you are skipping the sign in completely, unless something is not working as expected. Is it not working for you without this being disabled?

Reply:
Ah yup, not needed, thanks for catching!